app/views/admin/suspend_users/show.html.erb

```diff
@@ -6,21 +6,15 @@
       To suspend users for terms of service violations, please enter IDs of those users separated by comma or newline.
     </header>
 
-    <figure class="code">
```

Comment:
SuspendUsers also has been migrated to React, Not sure why it is still there. Let me know if I can remove it.